Workbooks/Workloads/MySQL Insights/MySQL Insights.workbook

```diff
@@ -79,7 +79,7 @@
             "delimiter": ",",
             "query": " Resources\n| where id in~ ({Resources})\n| project ['Virtual machine'] = tolower(id), Name = name\n| join kind = inner (Resources\n    | where type =~ 'microsoft.compute/virtualmachines/extensions' and name =~ 'workload.linuxconfigagent'\n    | project Vm = extract(@'(.+)/extensions/.+', 1, tolower(id)), Insight = tostring(tags[\"monitoring-insight\"])\n    | where Vm != '' and Insight == '{WorkloadTag}'\n    ) on $left.['Virtual machine'] == $right.Vm\n| project Name\n",
             "crossComponentResources": [
-              "{Subscription}"
+              "value::all"

```

Comment:
Shouldn't this be in just the Subscriptions they had picked?

Reply:
> Good point. Fixed this and OnboardedComputerCount